package.json

```diff
@@ -80,10 +82,16 @@
   "devDependencies": {
     "@sentry/browser": "^5.0.5",
     "@sentry/cli": "^1.40.0",
+    "@testing-library/cypress": "^8.0.7",
```

Comment:
Have you checked the `package-lock.json` to see if all new lines there are dependencies/subdependencies from those new packages and added only for dev?

Reply:
I double-checked the changes to the `package-lock.json and added a new section _Lib changes_ on this PR's description with my findings.

I believe all the added test dependencies are safe, and the other updates also.